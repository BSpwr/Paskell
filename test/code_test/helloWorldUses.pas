program helloWorldUses;
uses testFunctions;

begin
    hello_world;
    Writeln('hidden number is ', hidden_number);
    writeln(num); // This line should print an error message

end .