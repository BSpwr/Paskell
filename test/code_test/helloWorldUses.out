Hello, world.
hidden number is 4020
Identifier -> num <- is not defined.